### DISCUSSION TITLE
Custom Button

### DISCUSSION THREAD

**Discussion by uzproblue** [firecmsco/firecms#241] [Q&A]
Is there any solution to add a custom button with specific functionality in the same line as Save and Cancel buttons?

**Comment by fgatti675** [comment]
Hi @uzproblue 
This is not possible at the moment. 
You can try implementing it yourself and submit a PR or maybe you can reach us at hello@camberi.com to discuss possible ways forward.
As an alternative, you can add an additional column in the table view, and include a button that performs the action you need in the entity.
